### DISCUSSION TITLE
Large adif import times out

### DISCUSSION THREAD

**Discussion by pcpackrat** [magicbug/Cloudlog#2997] [Q&A]
I have my max_execution_timeout set to 300 but the page still times out at 30 seconds and becomes unresponsive for about 4 minutes.

ANy suggestions on what to tweak?  This is happening with LOTW imports too.

-Michael N5ZR

**Comment by magicbug** [comment]
Can you provide some info about what your running cloudlog on, hardware and software 

Also how many qsos are being held in cloudlog

**Comment by pcpackrat** [comment]
Debian 12, running as a VM on ESXi  6.7
VM has 2 CPU and 8GB RAM

When I import, I watch the memory go up to 50% and hangout until the process is supposedly done.

It does eventually import all the data but the site is locked up until its done.

-Michael

**Comment by magicbug** [comment]
Are you using Apache or Nginx? PHP-FPM? you might also find some useful hints in the error logs of the server
